docs/querying/sql-functions.md

```diff
@@ -28,17 +28,39 @@ sidebar_label: "All functions"
  This document describes the SQL language.
 :::
 
+This page provides a reference of all Druid SQL functions in alphabetical order. The **Learn More** at the end of each function section provides further documentation.
```

Comment:
```suggestion
This page provides a reference of Druid SQL functions in alphabetical order.
<-- The **Learn More** at the end of each function section provides further documentation. -->
For more detail on a function, refer to the following:
[bulleted list of the reference docs]
```
Let's leave the information about learn more out until we're done. It doesn't quite apply to the whole topic yet.
Also, let's remove "all" here and in the title.

Reply:
Line 31 should say "Apache Druid&circledR; SQL functions" since it's the first time we mention Druid in this topic (not counting the admonition).